koji_containerbuild/cli.py

```diff
@@ -73,6 +72,9 @@ def handle_container_build(options, session, args):
                       help=_("Perform a scratch build"))
     parser.add_option("--isolated", action="store_true",
                       help=_("Perform an isolated build"))
+    parser.add_option("--arches", dest='arch_override',
```

Comment:
Should this be defined similarly to how `yum_repourls` is done? By using `action='append'`
If so, we'd need to change how arch_override is read. 
Not sure how the regular `build` subcommand implements this.

Reply:
getArchList expects a list of strings. I think '--arches x86_64 powerpc64le' makes more sense than '--arches x86_64 --arches powerpc64le' but I'll adjust the code to pass a list like expected.

Reply:
now supports both --arches arch1, arch2 and --arches arch1 --arches arch2.

Fixed.

Reply:
I think it might require: `--arches 'one two'` which seems to be very awkward.

Reply:
A custom parser seems overkill IMO...
Anyways, I'll approve but won't merge to give others a chance to take another look.

Reply:
I'm pretty sure it works out to --arches arch1 arch2 but if I got it wrong, I can strip it back to --arches arch1 --arches arch2 which is very easy.

Reply:
The idea is that this works in the same way as it does for RPMs.
```
$ fedpkg build --help
[...]
  --arches [ARCHES [ARCHES ...]]
                        Build for specific arches
[...]
```

Reply:
Fixed.

Reply:
I still see the custom parser. Is that intended?

From `pyrpkg/cli.py::cliClient::register_build_common` in rpkg, it looks like this would do it:
```
self.build_parser_common.add_argument(                      
    '--arches', nargs='*', help='Build for specific arches')
```

Reply:
rkpg uses argparse, which supports parsing multiple arguments.
koji_containerbuild uses optparse, which does not: https://docs.python.org/2.6/library/optparse.html#callback-example-6-variable-arguments

Reply:
fair enough